ui/src/app/pages/admin/admin-institution-edit.tsx

```diff
@@ -672,15 +618,15 @@ export const AdminInstitutionEdit = fp.flow(withNavigation, withRouter)(class ex
                 onBlur={v => this.setState(fp.set(['institution', 'displayName'], v.trim()))}
             />
             <div style={{color: colors.danger}} data-test-id='displayNameError'>
-              {errors && errors.displayName}
+              {institution.displayName && errors && errors.displayName}
```

Comment:
Don't show a "can't be blank" error here on initial load in Add mode.  (Save is still blocked until this is not blank)